buildenv/jenkins/common/pipeline-functions

```diff
@@ -555,4 +586,99 @@ def generate_test_jobs(TARGET_NAMES, SPEC, ARTIFACTORY_SERVER, ARTIFACTORY_REPO)
     build job: 'Test_Job_Auto_Gen', parameters: parameters, propagate: false
 }
 
+def setup_pull_request() {
+    // Parse Github trigger comment
+    // Jenkins test sanity <platform>*
+    // Jenkins test extended <platform>*
+    // Jenkins compile <platform>*
+    // Jenkins test <comma_separated_test_target> <platform> *
+    //params.TESTS_TARGETS = params.ghprbCommentBody.substring(?)
+    /*
+    * Comment indexes - '+' is an offset of 0/1 if running tests or not
+    * 0 - Jenkins
+    * 1 - test / compile
+    * 1+ - test target
+    * 2+ - platforms
+    * 3+ - jdk versions
+    */
+    def PARSED_COMMENT = params.ghprbCommentBody.toLowerCase().tokenize(' ')
+    if (PARSED_COMMENT[0] != 'jenkins') {
```

Comment:
We should use a different trigger than 'jenkins' at least temporarily, so we can run pipeline PR builds concurrently with the old style PR builds for testing purposes. One suggestion is 'pipeline'

Reply:
Removed this check since it is technically redundant. The trigger regex in the jenkins job verifies PARSED_COMMENT[0]. We can change it to pipeline on the new jobs until we are ready to change over completely.